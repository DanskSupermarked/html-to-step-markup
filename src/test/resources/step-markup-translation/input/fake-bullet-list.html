<p><b>Klassisk havemøbelsæt i granit og Viro polyrattan</b><br /> Havemøbelsæt bestående af 1 bord samt 6 stole.<br /> Saint Tropez bordet er 100 x 205 cm og har et stel i galvaniseret stål og en bordplade med 3 granitplader.<br /> Ventura stolene har justerbar ryg og er lavet i sort Viro polyrattan i 4by4 flet.<br /> Saint Tropez sættet med 6 Ventura stole passer perfekt til den moderne familie, eller til par, som ofte har flere gæster<br /> Viro polyrattan er yderst nemt at vedligeholde, så hvis man foretrækker et minimum af vedligeholdelse, så er dette sæt det helt rigtige.<br /> <br /> <br /> •	Sæt med 6 stole<br /> •	Stole med justerbar ryg<br /> •	Klassisk og eksklusivt granitbord til hele familien på 100 x 205 cm<br /> •	Viro Polyrattan giver et minimum af vedligeholdelse<br /> <br /> <br /> Viro polyrattan: Det er et plastbaseret materiale, der betragtes som fremtiden inden for havemøbelproduktion. Materialet er UV-bestandigt og modstandsdygtigt overfor naturens luner. Rengøres let med vand og sæbe.<br /> <br /> <br /> Granit: Granit er et meget holdbart materiale, som kan modstå al slags vejr. Granit er ikke menneskeskabt men derimod hugget direkte ud af bjerge i Fjernøsten, og derfor er to møbler aldrig ens. Struktur og farven vil variere fra bord til bord.<br /> <br /> <br /> Saint Tropez sæt 205 cm. med 6 Ventura stole er altså et klassisk havemøbelsæt til familien eller parret, der foretrækker et minimum af vedligeholdelse og et eksklusivt design.</p>
