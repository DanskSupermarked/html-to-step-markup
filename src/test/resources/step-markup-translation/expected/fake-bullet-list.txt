<bold>Klassisk havemøbelsæt i granit og Viro polyrattan</bold>
Havemøbelsæt bestående af 1 bord samt 6 stole.
Saint Tropez bordet er 100 x 205 cm og har et stel i galvaniseret stål og en bordplade med 3 granitplader.
Ventura stolene har justerbar ryg og er lavet i sort Viro polyrattan i 4by4 flet.
Saint Tropez sættet med 6 Ventura stole passer perfekt til den moderne familie, eller til par, som ofte har flere gæster
Viro polyrattan er yderst nemt at vedligeholde, så hvis man foretrækker et minimum af vedligeholdelse, så er dette sæt det helt rigtige.
<bulletlist>
<bullet>Sæt med 6 stole</bullet>
<bullet>Stole med justerbar ryg</bullet>
<bullet>Klassisk og eksklusivt granitbord til hele familien på 100 x 205 cm</bullet>
<bullet>Viro Polyrattan giver et minimum af vedligeholdelse</bullet>
</bulletlist>
Viro polyrattan: Det er et plastbaseret materiale, der betragtes som fremtiden inden for havemøbelproduktion. Materialet er UV-bestandigt og modstandsdygtigt overfor naturens luner. Rengøres let med vand og sæbe.

Granit: Granit er et meget holdbart materiale, som kan modstå al slags vejr. Granit er ikke menneskeskabt men derimod hugget direkte ud af bjerge i Fjernøsten, og derfor er to møbler aldrig ens. Struktur og farven vil variere fra bord til bord.

Saint Tropez sæt 205 cm. med 6 Ventura stole er altså et klassisk havemøbelsæt til familien eller parret, der foretrækker et minimum af vedligeholdelse og et eksklusivt design.
